### ISSUE TITLE
attempt to perform arithmetic on a nil value ( field '?')

### ISSUE THREAD

**Issue by Beta-Bella** [InZidiuZ/LegacyFuel#101]
Hello dear Dev.

First of all i love you script. And we never had any issues. 

Since we switched over to Ox-inventory. 
Our first issue is , when we have the jerrycan as an item - We can use it to refill the car but after 2 seconds, it stops and you cannot fuel the car.  There is no error shown in the client nor serversided console sadly. 

And the other is - as shown on the picture on this thread. 
I havent managed to reproduce it yet since people are not willing to give enough information what they do - and what not. 

We run on the latest ESX legacy with Ox-inventory and its modules. 

lovely greetings. 

![image](https://github.com/InZidiuZ/LegacyFuel/assets/128776783/58cf600a-4e84-4d4c-aeb8-3d448ac249c3)

**Comment by Feurfelix2409** [comment]
Dear BellaStreet,

I use this script too and have the same Problem with the Jerrycan. Can you please tell me how you fixed it?

 Thanks in advance

**Comment by Beta-Bella** [comment]
in the client.lua i believe , Line 159 TriggerEvent('fuel:startFuelUpTick', true, ped, vehicle) 

It was on false with us ( why  we dont know, we switched it to true) but it gives your Jerrycan infinite ammo. So they basically can missuse it. 

and we had to switch to weapon_petrolcan ( since we use oxinventory) 


We switched to a different system while we tried to fix it somehow. 

> Dear BellaStreet,
> 
> I use this script too and have the same Problem with the Jerrycan. Can you please tell me how you fixed it?
> 
> Thanks in advance
